autoload/sy/repo.vim

```diff
@@ -645,6 +651,7 @@ let s:default_vcs_cmds_diffmode = {
       \ 'accurev':  'accurev cat %f',
       \ 'perforce': 'p4 print %f',
       \ 'tfs':      'tf view -version:W -noprompt %f',
+      \ 'jj':       'jj cat -- %f',
```

Comment:
Shouldn't this be getting the file contents from `@-` (ie, `jj cat -r @- -- %f`)?

The `jj diff` in default_vcs_commands shows the diff between `@-` and `@`, and practically speaking, `@-` is jj's `HEAD` analogue.

It's more complicated if `@` is a merge, since that would involve synthesising a merged version of the parents' files like `jj diff` does. I don't have a good solution for that, unfortunately.